warehouse/accounts/__init__.py

```diff
@@ -124,6 +124,12 @@ def includeme(config):
     config.register_service_factory(
         TokenServiceFactory(name="email"), ITokenService, name="email"
     )
+    config.register_service_factory(
```

Comment:
NTS: These can be removed now that we're no longer stashing any 2FA/TOTP state in cookies.